src/features/events/store.ts

```diff
@@ -14,16 +14,45 @@ import {
   ZetkinLocation,
 } from 'utils/types/zetkin';
 
+export enum ACTION_FILTER_OPTIONS {
+  MISSING = 'Contact person missing',
+  UNSENT = 'Unsent notifications',
+  PENDING = 'Signups pending',
+  UNDERBOOKED = 'Underbooked',
+  OVERBOOKED = 'Overbooked',
```

Comment:
The values here are adapted for humans, which is a red flag for me. And much as I expected, they are not used anywhere, instead the key (lowercase) is used. Let's just use the lowercase strings as values instead.

```suggestion
  MISSING = 'missing',
  UNSENT = 'unsent',
  PENDING = 'pending',
  UNDERBOOKED = 'underbooked',
  OVERBOOKED = 'overbooked',
```